schema_source/jetstream/api/v1/definitions.json

```diff
@@ -5,6 +5,33 @@
   "description": "Shared definitions for the JetStream API",
   "type": "object",
   "definitions": {
+    "golang_duration_nanos": {
+      "$comment": "nanoseconds depicting a duration in time",
```

Comment:
can you change the comment to "nanoseconds depicting a duration in time, signed xx bit integer"

Reply:
fixed this, also put in some `sed` hacks to fix these numbers :( go would panic on parsing the ones node produce, this really is a mess.